frontend/src/components/CalcSpectrumPlot.tsx

```diff
@@ -81,6 +105,7 @@ const CalcSpectrumPlot_ = ({
               type: "linear",
               fixedrange: false,
             },
+            updatemenus: updatemenus,
```

Comment:
```suggestion
            updatemenus,
```
this isn't important but it's some syntactic sugar you can use if the property name equals the variable name